Testreglar/1.1.1/Nett/nett-1.1.1f.json

```diff
@@ -0,0 +1,129 @@
+{
+	"namn": "Nett-1.1.1f Tidsbaserte medier har tekstalternativ 2023",
+	"id": "nett-1.1.1f",
+	"testlabId": 508,
+	"versjon": "1.0",
+	"type": "Nett",
+	"spraak": "nb",
+	"kravTilSamsvar": "<p>Ikke-tekstlig innhold som er tidsbaserte medier, har kort tekstalternativ med en beskrivende identifikasjon av det tidsbaserte mediet.</p>",
```

Comment:
```suggestion
	"kravTilSamsvar": "<p>Video og lydklipp har et kort tekstalternativ med en beskrivende identifikasjon av det tidsbaserte mediet.</p>",
```

Reply:
Foreslår å heller bruke Video og lydklipp siden det er det som faktisk blir spurt om i testregelen.

Reply:
I spørsmål 3.2 spør vi om video eller lyd er direktesendt. Dersom vi tolker at kravet til tekstalternativ (1.1.1) bare gjelder for _forhåndsinnspilt innhold_ **må det går frem i kravet til samsvar**.